lambdas/death-minimisation/src/test/java/uk/gov/di/data/lep/GroDeathNotificationMinimisationTest.java

```diff
@@ -69,6 +78,7 @@ void minimiseGroDeathEventDataReturnsMinimisedDataWithNoEnrichmentFields() {
 
     @Test
     void minimiseGroDeathEventDataReturnsMinimisedDataWithPartialEnrichmentFields() {
+        when(context.getLogger()).thenReturn(logger);
```

Comment:
I think we should put this in a before all, and then we can verify its called on all of these easily